### PR TITLE
WIP: Initial documentation structure

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
This PR introduces a structure into the documentation, namely the following sections:
- Docs: Reference material for using OpenFeature e.g. the specific commands and code
- Getting Started: Guide on using OpenFeature 
- Tutorials: Use Cases and specific implementations 
- Explanations: Extensive explanations of the different components 

TODO:
[] Getting Started Guide

Design:
[] Dark mode does not look nice atm

### CODE REVIEW

File: docs/getting-started/overview.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+# Getting Started with Open Feature
+
+## FlagD
+
+TODO
+
+## Open Feature Operator
+
+TODO
```
Comment (toddbaert) [inline docs/getting-started/overview.md:14]: White flagd and OFO are important, I think the top section should be for using the OpenFeature SDK in-process to evaluate flags, that's are primary use case.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+This repository provides the documentation for OpenFeature. 
+
+### Documentation Structure
+
+We are following a slgithly modified version of the [diataxis framework](https://diataxis.fr/) for our documentation:
+
+- Docs: Reference material for using OpenFeature e.g. the specific commands and code
+- Getting Started: Guide on using OpenFeature 
+- Tutorials: Use Cases and specific implementations 
+- Explanations: Extensive explanations on the different components
```
Comment (toddbaert) [inline README.md:12]: Thanks for this explanation.

File: docs/getting-started/overview.md (hunk 1 above)
Comment (AnaisUrlichs) [reply docs/getting-started/overview.md:14]: I guess we would want to have three different getting started guides then?

Comment (toddbaert) [reply docs/getting-started/overview.md:14]: I think so... Often, OFO and Flagd will be used together, but not always. We will clarify in our meeting.

File: docs/getting-started/overview.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 1
+id: overview
+---
+
+# Getting Started with Open Feature
+
+## FlagD
```
Comment (toddbaert) [inline docs/getting-started/overview.md:8]:
```suggestion
## OpenFeature SDK

## FlagD
```

**Review by toddbaert — APPROVED** [review]

**Pull request merged**